scripts/lib.star

```diff
@@ -497,6 +499,26 @@ def build_frontend_step(edition, is_downstream=False):
         ],
     }
 
+def build_frontend_docs(edition, lint=False):
+    if edition == 'enterprise':
+        return None
+
+    if lint:
+        script = './scripts/ci-reference-docs-lint.sh'
+    else:
+        script = './scripts/ci-reference-docs-build.sh'
+
+    return {
+        'name': 'build-frontend-docs',
+        'image': build_image,
+        'depends_on': [
+            'build-frontend'
```

Comment:
Could this depend on 'test-frontend' instead, so it can run in parallel with build-frontend? Otherwise it will slow down CI.

Reply:
It cannot, we need to get the packages built before continue with this one. The only way we can do that is to move the build packages to its own step instead of having it as part of the build-frontend.